cpp/perspective/CMakeLists.txt

```diff
@@ -352,13 +352,22 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
 			message("${Red}Manylinux build has no python shared libraries${ColorReset}")
 			find_package( Python ${PSP_PYTHON_VERSION} REQUIRED COMPONENTS Interpreter )
 			find_package( PythonHeaders ${PSP_PYTHON_VERSION} REQUIRED)
+			message("${Cyan}Using PYTHON_INCLUDE_DIRS: ${PYTHON_INCLUDE_DIRS}, Python_LIBRARIES: ${Python_LIBRARIES} ${ColorReset}")
+
+			# find_package(PythonHeaders) sets the uppercased version of
```

Comment:
we wrote this cmake package, just modify it to match

Reply:
Done